app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(command: str) -> None:
+    current_file = command.split(" ")[1]
+    new_file = command.split(" ")[2]
```

Comment:
you can unpack these variables in the easiest way, also use the more descriptive names